### DISCUSSION TITLE
Keda 2.12.1 enforces minimum replica count of 1 despite documentation says default value is 0

### DISCUSSION THREAD

**Discussion by SanjayPapaiahgari25** [kedacore/keda#5809] [Q&A / Need Help]
Hi

I am facing an issue where even though i am setting the minReplicaCount to 0 in the keda deployment yamls for a kubernetes deployment, the change is not getting applied and the default minimum value it is setting is 1, is this an issue with keda 2.12.1 or am i missing something. I am new to keda, so any help would be appreciated. We cannot upgrade keda due to other reasons incase if this issue is specific to keda 2.12.1 and fixed in later versions.

**Comment by tomkerkhove** [comment]
Where in the documentation did it state to use 0 by default?

**Reply by SanjayPapaiahgari25** [reply]
It says that the default value for minReplicaCount if we don't specify the attribute is 0, you can refer it here under ScaledJob Spec section here in the comment
https://keda.sh/docs/2.12/concepts/scaling-jobs/#scaledjob-spec
